test/angular/getChanges.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { expect } from "chai";
+import { getChanges, Input } from "../../src/angular";
+import { Changes } from "../../src/types";
+
+describe("Angular getChanges", () => {
```

Comment:
This test still fails on my computer with this error message:

```
1) Angular getChanges
       static content:

      AssertionError: expected { Object (originTemplateReplacement, files) } to deeply equal { Object (originTemplateReplacement, files) }
      + expected - actual

       {
         "files": [
           {
             "content": "<button>hello world</button>"
      -      "path": "\\baseDir\\test\\test.component.html"
      +      "path": "/baseDir/test/test.component.html"
           }
         ]
         "originTemplateReplacement": "<app-test></app-test>"
       }

      at Context.<anonymous> (test\angular\getChanges.spec.ts:25:28)
      at processImmediate (node:internal/timers:464:21)
```

There seems to be an issue with the slashes in the path.

Probably it will fix the issue if we use the path package of node.js here:

Instead of `path: "/baseDir/test/test.component.html",` we should write `path.join('baseDir', 'test', 'test.component.html')`